prow/config.yaml

```diff
@@ -69,61 +85,21 @@ branch-protection:
       restrictions:
       repos:
         api:
-          protect: false
-          branches:
-            <<: *blocked_branches
-            master:
-              protect: true
+          branches: *blocked_branches
         istio.io:
-          protect: false
-          branches:
-            <<: *blocked_branches
-            master:
-              protect: true
+          branches: *blocked_branches
         operator:
```

Comment:
I  think the operator also needs the same blocked branches.

Reply:
Done